src/main/java/com/rultor/agents/ecs/Amazon.java

```diff
@@ -27,33 +27,34 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
  * OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package com.rultor.agents.ec2;
+package com.rultor.agents.ecs;
 
-import com.amazonaws.services.ec2.model.Instance;
+import com.amazonaws.services.ecs.model.Container;
 import com.jcabi.aspects.Immutable;
 import java.io.IOException;
 
 /**
- * Amazon EC2 abstraction.
+ * Amazon ECS abstraction.
  *
  * @author Yuriy Alevohin (alevohin@mail.ru)
  * @version $Id$
  * @since 2.0
- * @todo #629 Add implementation for com.rultor.agents.ec2.Amazon.
- *  It must create com.amazonaws.services.ec2.AmazonEC2 client
+ * @todo #629 Add implementation for com.rultor.agents.ecs.Amazon.
+ *  It must create com.amazonaws.services.ecs.AmazonECSClient
  *  with config params "credentials", "zone", "type", "key". Use
- *  client.runInstances(com.amazonaws.services.ec2.model.RunInstancesRequest)
- *  to run on-demand instance. Method runOnDemand must finally
+ *  client.runInstances
+ *  (com.amazonaws.services.ecs.model.RegisterContainerInstanceRequest)
```

Comment:
@original-brownbear Sorry for disturb, but can you give me any advice here? Looks like this puzzle must be totally refactored since I cannot find nothing that looks like `RunInstancesRequest` and `runInstances` in ECS.